src/components/python-pages/PythonLinkOutline.js

```diff
@@ -11,11 +11,34 @@ class PythonLinkOutline extends React.Component {
         e.target.style.color = "#365789";
     }
 
+    constructor() {
+        super();
+        this.state = {
+            lessons: [],
+        };
+    }
+
+  componentDidMount() {
+    fetch("https://adminbluebirdteaching.pythonanywhere.com/lessons/PCEP Course")
+      .then((response) => response.json())
+      .then((data) => {
+        this.setState({ lessons: data });
+      });
+  }
+
     render() {
+        const { lessons } = this.state;
         return (
             <div>
                 <ol className="list-medium">
                     <li><Link to="/python/introduction" style={{ color: "#365789" }} onMouseOver={this.changeLinkColorEnter} onMouseOut={this.changeLinkColorLeave}>How To Code In Python</Link></li>
+                    {lessons.map((lesson, index)  => (
```

Comment:
When you tested this, which lesson appeared first?  When you're mapping through, did the newest lesson appear first, or did the oldest lesson appear first?  Just double checking.

Reply:
Currently, the oldest lesson appears first, as seen below:

![image](https://github.com/YomenT/bluebird-teaching-frontend/assets/94801110/5086c15d-1db4-4e69-a3bb-56bd0377b36c)
![image](https://github.com/YomenT/bluebird-teaching-frontend/assets/94801110/1872644d-14d2-47b7-a5fe-7a1d963ad03b)